R/article.R

```diff
@@ -175,6 +175,19 @@ frontiers_article <- function(..., keep_tex = TRUE) {
   pdf_document_format("frontiers", keep_tex = keep_tex, ...)
 }
 
+#' @section \code{jasa_article}: Format for creating submissions to the
+#'   Journal of the Acoustical Society of America. Adapted from
+#'   \url{https://acousticalsociety.org/preparing-latex-manuscripts/}.
+#' @export
+#' @rdname article
+jasa_article <- function(
+  ..., keep_tex = TRUE, latex_engine = "xelatex"
+) {
+  pdf_document_format(
+    "jasa", keep_tex = keep_tex, latex_engine = latex_engine, ...
+  )
+}
```

Comment:
Based on https://github.com/rstudio/rticles/pull/364#issuecomment-756005199
we could opt for a solution to do this
```suggestion
jasa_article <- function(
  ..., keep_tex = TRUE, latex_engine = "xelatex"
) {
  if ( !"revtex4-1" %in% tinytex::tl_pkgs()) tinytex::tlmgr_install("revtex4-1")
  pdf_document_format(
    "jasa", keep_tex = keep_tex, latex_engine = latex_engine, ...
  )
}
```

Otherwise we need to include the cls file directly in the skeleton bundle. 

@yihui what is your thoughts on that ? Is it ok to do this ? 
[Long Story](https://github.com/rstudio/rticles/pull/364#pullrequestreview-562892304) put short here: 
`JASA.cls` is based on another cls which is in a package `revtex4-1`. First thing they do in JASA.cls is to test if this package is there and errors otherwise. That means we can't rely on **tinytex** auto-installation (unless we had their custom message in the known text to parse). To compile `revtex4-1` must be found before any rendering. So we need to make it available. 
Thank you.

Reply:
I would do that in the `setup` chunk of `skeleton.Rmd` instead:

```r
if (tinytex:::is_tinytex() && !tinytex:::check_installed('revtex4-1')) {
  tinytex::tlmgr_install("revtex4-1")
}
```

to keep the output format function a little cleaner (with fewer side-effects).

Reply:
It will be less magic for users, but I guess all **rticles** users know about latex. 
And with the proper comment to explain this package is required by the template, this would be indeed enough. 

@stefanocoretta lets do this then ! 

Thanks @yihui!